Makefile

```diff
@@ -1,5 +1,5 @@
-PYTEST_OPTIONS = -vvv
-CLOUDIGRADE_USER=''
+PYTEST_OPTIONS :=-vvv
```

Comment:
Any specific reason for using non recursive variables?

Reply:
No really I was fiddling around thinking about indicating a default driver and then changed my mind.

Reply:
got rid of changes since they were not needed